platformcomponents/desktop/attachment-file-preview-thumbnail.json

```diff
@@ -0,0 +1,12 @@
+{
+  "attachment-file-preview-thumbnail": {
+    "sent": {
+      "comment": "Applied to file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=29250%3A45485",
+      "border": "@theme-outline-secondary-normal",
+      "icon": "@theme-common-text-white",
+      "icon-background": "@theme-background-accent-normal",
+      "overlay": "@theme-common-overlay-primary"
+    }
```

Comment:
It is missing here because it's already present in the UIToolkit's existing tokens. I could add it here also if you think is should be part of this component.

Reply:
yes please, as the design team can come and change the color of the text assigned to this specific component, and that would bring problems later :)